projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -106,6 +107,23 @@ const RecordMeterBar: React.FC< RecordMeterBarProps > = ( {
 					} ) }
 				</ul>
 			</div>
+			<table className="screen-reader-text">
+				<caption>{ __( 'Summary of the records', 'jetpack' ) }</caption>
+				<tbody>
+					<tr>
+						<th scope="col">{ __( 'Record type', 'jetpack' ) }</th>
```

Comment:
As this table will be not be displayed but read, let's go for maximum clarity here.

Let's have a row of headers, then a row for each post type.

Record type  | Record count   <-- header with scope col
Record type 1 | 123
Record type 2 | 234

Reply:
Okay yeah I totally agree with this. A job for tomorrow I think, as I'm about to head off. but I like the maximum clarify approach and will adjust this in the AM.

Reply:
Okay have shuffled the tables around, with a nice byproduct of one less `map` too, which is always nice. Hopefully things are a bit tidier / max clarity!